pkg/certs/cert-inspection/certgraphutils/util.go

```diff
@@ -50,9 +51,9 @@ func (n ConfigMapLocationByNamespaceName) Less(i, j int) bool {
 	return n[i].Name < n[j].Name
 }
 
-func LocateCertKeyPair(targetLocation certgraphapi.InClusterSecretLocation, certKeyPairs []certgraphapi.PKIRegistryInClusterCertKeyPair) (*certgraphapi.PKIRegistryInClusterCertKeyPair, error) {
+func LocateCertKeyPair(targetLocation certgraphapi.InClusterSecretLocation, certKeyPairs []certgraphapi.PKIRegistryInClusterCertKeyPair, nodes map[string]int) (*certgraphapi.PKIRegistryInClusterCertKeyPair, error) {
 	for i, curr := range certKeyPairs {
-		if targetLocation == curr.SecretLocation {
+		if certgraphanalysis.FormatSecretLocation(targetLocation, nodes) == curr.SecretLocation {
```

Comment:
This change doesn't look correct to me.  The only thing that should return for "matches" should be an exact location match.  The failures we're seeing mean that we're not describing that location properly.

Revisions should be removed (that's https://github.com/openshift/origin/pull/28362) and the node name and IP should be replaced by <master-0> <IP-master-0-> and the like when we store the raw TLS info.  Probably in the serialization step, with a straight sed

Reply:
That would make raw TLS info in fact processed, not "raw", isn't it? My understanding was that rawTLSInfo produced in cluster would be actual state of the cluster - all revisions, hashes and IPs preserved. This would be useful for recert tool, which has work with actual cluster secrets directly.

However, ownership JSON would have preprocessed names - no revisions/hashes, synthetic names for certs containing IPs. The test would convert actual name using `FormatSecretLocation` and use this name to compare with ownership entries (I'll rename vars do use better names here)